src/pybind/mgr/dashboard/frontend/e2e/block/images.e2e-spec.ts

```diff
@@ -30,4 +32,37 @@ describe('Images page', () => {
       expect(images.getTabText(2)).toEqual('Overall Performance');
     });
   });
+
+  describe('create, edit, & delete image test', () => {
+    beforeAll(() => {
+      images.navigateTo();
+    });
+
+    it('should create a pool for testing purposes', () => {
+      pools.navigateTo('create');
+      pools.create('056rq', 8, 'rbd').then(() => {
```

Comment:
Those vars would be used here:
```suggestion
      pools.create(poolName, 8, 'rbd').then(() => {
```